src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -86,7 +86,9 @@ const Conversation: React.FC<ConversationProps> = props => {
   }
 
   const messageGroups = groupMessages(
-    conversation.messagesConnection.edges.map(edge => edge.node)
+    conversation.messagesConnection.edges
+      .map(edge => edge.node)
+      .filter(node => node.body.length > 0)
```

Comment:
very nice catch 🥇

Reply:
Is there ever the chance where `edge.node` will return `null`?

Reply:
(This is a nice add-a-test example.)